src/routes/chains.rs

```diff
@@ -0,0 +1,59 @@
+use crate::cache::cache_operations::{CacheResponse, RequestCached};
+use crate::config::{
+    base_config_service_url, chain_info_cache_duration, chain_info_request_timeout,
+};
+use crate::providers::info::{DefaultInfoProvider, InfoProvider};
+use crate::utils::context::Context;
+use crate::utils::errors::ApiResult;
+use rocket::response::content;
+
+/**
+ * `/v1/chains/<chain_id>/` <br/>
+ * Returns [ChainInfo](crate::models::chains::ChainInfo)
+ *
+ * # Balances
+ *
+ * This endpoint returns the [ChainInfo](crate::models::chains::ChainInfo) for a given `chainId`
+ *
+ * ## Path
+ *
+ * - `/v1/chains/<chain_id>/`returns the `ChainInfo` for `<chain_id>`
+ *
+ */
+#[get("/v1/chains/<chain_id>")]
+pub async fn get_chain(context: Context<'_>, chain_id: String) -> ApiResult<content::Json<String>> {
```

Comment:
Regarding the type of `chain_id` if it makes things easier for validation on the CGW we can assume this to be a Big Int. This is how it's represented internally in the safe-config even though it's exposed as a string (this is how the framework maps a BigInt to a Json field). 

But if this is mostly used for the path I'd say it should be fine – I don't think we need to validate it. Eg.: giving as a `chain_id = foo` should return a `404` from the config service and the CGW would propagate that.

Reply:
Actually, there is one place where I wish it was just a plain number, which is when we calculate the `safeTxHash` but we just parse it:

https://github.com/gnosis/safe-client-gateway/blob/34e751b096b108174be8b4f8c4d8d0c2475f1433/src/utils/transactions.rs#L84

it's either way fine :) I don't see this as a solid argument against having it as a string :thinking: In fact, I am not sure if `chain_id`s are guaranteed to always be numbers, maybe at some point people will start putting all sorts of chars there :thinking:

Reply:
I was looking at EIP155 https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md

And there it's used with `{0,1} + CHAIN_ID * 2 + 35` so I assume it should only be represented as a number. But this was more to check if on your side it'd make things easier as I'd say we can assume it as a number (else we can just use string) 👍

Reply:
Ah that's good to know! If there is also a chance that the `chain_id` could potentially be a big number, which could trip up our (de)serializers too :thinking: Let's keep it as a `String` as it seems like the option with the fewest assumptions on the values :)

Reply:
Internally in the DB it's also `uint64` which I think it's ok given the current values that we see in the chains 👍 It's serialised as `String` so even if we increase the size of the parameter in the DB there are no changes in the API contract